data/brands/amenity/bar.json

```diff
@@ -42,6 +42,19 @@
         "name": "Coyote Ugly"
       }
     },
+    {
+      "displayName": "Pijalnia Wódki i Piwa",
+      "id": "pijalniawodkiipiwa-ca2c56",
+      "locationSet": {"include": ["pl"]},
+      "tags": {
+        "amenity": "bar",
+        "brand": "Pijalnia Wódki i Piwa",
+        "brand:website": "http://pwip.com.pl",
```

Comment:
this can be fetched from ```brand:wikidata``` so I don't think this is needed

Reply:
I removed `brand:wikipedia` and `brand:website`
I added website to wikidata item